<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SolveDlg</class>
 <widget class="QDialog" name="SolveDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Cube Solve Progress</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="..\resources.qrc" >:/images/HappySolver.png</iconset>
  </property>
  <widget class="QPushButton" name="resetButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>90</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string>Reset</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="goButton" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text" >
    <string>Go</string>
   </property>
   <property name="autoDefault" >
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>68</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text" >
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="" >
   <property name="geometry" >
    <rect>
     <x>117</x>
     <y>14</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>Solutions:</string>
   </property>
  </widget>
  <widget class="QLabel" name="solutions" >
   <property name="geometry" >
    <rect>
     <x>181</x>
     <y>14</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <family>MS Shell Dlg</family>
     <pointsize>8</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QLabel" name="status" >
   <property name="geometry" >
    <rect>
     <x>50</x>
     <y>70</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text" >
    <string>TextLabel</string>
   </property>
   <property name="textFormat" >
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxStats" >
   <property name="geometry" >
    <rect>
     <x>110</x>
     <y>34</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title" >
    <string>Run Statistics</string>
   </property>
   <widget class="QLabel" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>18</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Changes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>58</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>CPU time:</string>
    </property>
   </widget>
   <widget class="QLabel" name="" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>38</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QLabel" name="changes" >
    <property name="geometry" >
     <rect>
      <x>62</x>
      <y>18</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
    <property name="textFormat" >
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CPUTime" >
    <property name="geometry" >
     <rect>
      <x>62</x>
      <y>58</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>TextLabel</string>
    </property>
    <property name="textFormat" >
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="speed" >
    <property name="geometry" >
     <rect>
      <x>58</x>
      <y>38</y>
      <width>85</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text" >
     <string>&lt;font color="#CF1111">TextLabel</string>
    </property>
    <property name="textFormat" >
     <enum>Qt::RichText</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="../resources.qrc" />
 </resources>
 <connections/>
</ui>
